Formula/mosdepth.rb

```diff
@@ -1,13 +1,12 @@
 class Mosdepth < Formula
   desc "Fast BAM/CRAM depth calculator"
   homepage "https://github.com/brentp/mosdepth"
-  url "https://github.com/brentp/mosdepth/releases/download/v0.2.3/mosdepth"
-  sha256 "0aaeb283b296e5ee0a2d819b4842b92289145ed17a37210144f0e02edd3629c4"
+  url "https://github.com/brentp/mosdepth/releases/download/v0.2.9/mosdepth"
+  sha256 "a73283fb1a7465601a4d2d738f6f832f2fd84bf9181e0d4d2b91453da385177c"
 
   bottle do
     root_url "https://linuxbrew.bintray.com/bottles-bio"
     cellar :any_skip_relocation
-    sha256 "bb05121fdf75c04bc21a7ea46008eb3309a4d9f52ccbba0bde202605a77f3f28" => :x86_64_linux
```

Comment:
No need to remove the bottle.

Reply:
Oh, I dunno if I did that, I used brew bump-formula-pr with --url and --share256 args. It looks like that did it. I can restore?

Reply:
Weird. Yes, please. You can make the changes in your editor `brew edit mosdepth` and use `git commit --amend` to modify the commit and `git push --force cmdcolin mosdepth-0.2.9` to update the PR.